src/components/ForgotPassword/ForgotPassword.styles.js

```diff
@@ -0,0 +1,53 @@
+import styled, { keyframes } from 'styled-components';
+import colors from '../styles/colors';
+
+const bouncedIn = keyframes`
```

Comment:
Maybe we should have the animation in a separate file, since the same animation is used on other screens.
This way we could reuse it.

Reply:
Ok thanks!

Reply:
ready